src/test/utils/VaultUtils.sol

```diff
@@ -74,6 +80,42 @@ abstract contract BasicVaultTest is UniswapV2Fixture {
         assertEq(getToken0DepositRequests(), amount);
     }
 
+    function test_depositToken0DepositDisabled() public {
+        vm.prank(pauser);
+        vault.pause();
+        vm.prank(strategist);
+        vault.setDepositsDisabled();
+        vm.prank(pauser);
+        vault.unpause();
+        vm.stopPrank();
```

Comment:
we can remove this since `vm.prank()` only pranks the next call. `vm.stopPrank` is relevant when we use `vm.startPrank` because then it applies to all the external calls until `stopPrank` is used